sanic/app.py

```diff
@@ -454,6 +454,13 @@ def websocket(
         def response(handler):
             async def websocket_handler(request, *args, **kwargs):
                 request.app = self
+                if not getattr(handler, "__blueprintname__", False):
```

Comment:
I actually like the if/else for clarity of reading. I think it's a simple enough logic branch that it doesn't necessarily require moving it to its own method... with that being said, perhaps it's worth revisiting if we do end up adding more complexity here.

Reply:
@sjsadowski Looking back at the comment, I see how my message was a bit confusing. I suggested the refactoring because we can re-use what we do in and around line#457 and carry it over to line#898